TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleOpMain.java

```diff
@@ -48,7 +49,6 @@
     //TODO: Add servo stuff here
     public GamepadEx m_driverOp, m_toolOp;
     private Button shooterStart, shooterStop, dpadUp, dpadDown, goalLift, toggleIntake;
-
     @Override
     public void initialize() {
```

Comment:
Change the ConditionalCommand binding to `whileHeld()` instead of `whenPressed()`